index.rst

```diff
@@ -17,6 +17,7 @@ Table of Contents
    simulator
    pulsar
    stingray/docs/api
+   tutorials
```

Comment:
The tutorials rst file is not supposed to be in here. It's deprecated.

Reply:
Ah ok. I'll take it away and add it to the ignored patterns

Reply:
Done.